src/operator/tensor/elemwise_unary_op.h

```diff
@@ -348,6 +351,40 @@ class UnaryOp : public OpBase {
       LogUnimplementedOp(attrs, ctx, inputs, req, outputs);
     }
   }
+
+#if MSHADOW_USE_MKL == 1
+#define MKLLOG(fname, DType) \
+static void MKLLog(size_t size, const DType* pIn, DType* pOut) { \
```

Comment:
Macro has the same name with function. Using macro here seems a little bit overkill for me. How about: 
```
#if MSHADOW_USE_MKL == 1
static inline void MKL_Log(int size, const float* pIn, float* pOut) {
  vsLn(size, pIn, pOut);
}

static inline void MKL_Log(int size, const double* pIn, double* pOut) {
  vdLn(size, pIn, pOut);
}
#endif
```

Also, please note that `vsLn/vdLn` take `MKL_INT` as the first input. `size_t` might overflow.

Reply:
Thanks, it's done